packages/tests/web-test-runner.config.js

```diff
@@ -164,8 +164,14 @@ export default {
   rootDir: '../',
   // Note this file list can be overridden by wtr command-line arguments.
   files: [
-    '../lit-html/development/**/*_test.(js|html)',
+    '../labs/observers/development/**/*_test.(js|html)',
+    '../labs/react/development/**/*_test.(js|html)',
+    '../labs/router/development/**/*_test.js',
+    '../labs/scoped-registry-mixin/development/**/*_test.(js|html)',
+    '../labs/ssr/development/**/*_test.(js|html)',
```

Comment:
I think that `ssr` won't work in this consolidated `tests/` configuration, because it needs an extra command-line flag, and it needs to register its special SSR plugin:

https://github.com/lit/lit/blob/3e181bcb3d969775eda799fd6fcae1ead843225b/packages/labs/ssr/package.json#L19

https://github.com/lit/lit/blob/3e181bcb3d969775eda799fd6fcae1ead843225b/packages/labs/ssr/web-test-runner.config.js#L20

So for now I would suggest not including `ssr` in this consolidated config, instead letting it run in its own standalone `wtr` invocation as before.

Reply:
this is similar / related to what i proposed in the original PR
I feel like we shouldn't have to choose here.

we should be able to run all tests locally if we want
and run all dev tests locally when dev tests fail
and run all prod tests locally when prod fails fail
and run all tests remotely
and run all dev tests remotely
and run all prod tests remotely

We need the ability to run all tests, and also isolate where we run tests to resolve blocks


I'd prefer the option to run all `dev` tests and `prod` tests

Reply:
Also SSR totally runs and passes so not sure what I'd be 'fixing' there

Reply:
> Also SSR totally runs and passes so not sure what I'd be 'fixing' there

Ah interesting. I think the reason it apparently passes must be that there are actually no files inside `labs/ssr/development`, so `wtr` is just not matching any test files there. We don't emit anything there; there is no prod vs dev distinction for SSR. 

```
> cd lit
> npm run build
> ls packages/labs/ssr/development
No such file or directory
```

Reply:
I think adding an `"integration": "cd packages/tests ..."` is a good start.

I'll be pushing for the ability to run tests based on mode in isolation in the future though. I feel like that's pretty standard and something we need to support _if_ we're gonna treat dev and prod mode as distinct

Reply:
We can add `NODE_OPTIONS=--experimental-vm-modules` to the main wtr script, yes?

Reply:
In addition to the `--experimental-vm-modules` flag, SSR also needs this separate middleware loaded: https://github.com/lit/lit/blob/3e181bcb3d969775eda799fd6fcae1ead843225b/packages/labs/ssr/web-test-runner.config.js#L18

I'm not sure what the implication of having that middleware loaded for all of our other packages would be.

So I was thinking for now, safer to just keep SSR separate, and consider consolidating separately?

Reply:
(for future brian, SSR wasn't even being run)

Reply:
I think we should remove the `ssr` glob from this list, even though we know it matches no files, just because it's a bit misleading.

Reply:
resolving because I think we agreed to let SSR test in the saucelabs job but not _in_ saucelabs browsers.